src/actions/tomoAction.js

```diff
@@ -50,10 +50,13 @@ export const instantiateContracts = () => async (dispatch, getState) => {
   console.log(listGame);
   let currentGameAddress = listGame[listGame.length - 1];
   const game = new web3.eth.Contract(GameArtifact.abi, currentGameAddress);
+  let amountQuestion = await game.methods.currentQuestion().call({ from });
```

Comment:
```suggestion
let questionCount
```
đặt tên biến cho đúng & dễ hiểu